libs/blocks/table/table.css

```diff
@@ -37,17 +37,25 @@
   margin-right: 8px;
 }
 
-.table.merch div[class*="col-"] {
+.table.merch .row-heading .col {
   border-top: 1px var(--border-color) solid;
+}
+
+.table.merch div[class*="col-"] {
```

Comment:
Does this need the extra specificity or could you just use the attribute, like `.table.merch [class*="col-"]`?